src/RuleSet/Sets/PERCS2x0Set.php

```diff
@@ -35,6 +35,10 @@ public function getRules(): array
         return [
             '@PER-CS1.0' => true,
             'concat_space' => ['spacing' => 'one'],
+            'method_argument_space' => [
+                'attribute_placement' => 'standalone',
+                'on_multiline' => 'ensure_fully_multiline',
+            ],
```

Comment:
Is it actually needed here? Both values are default values for these options. Probably it could be:

```suggestion
            'method_argument_space' => true,
```

This way it would reset custom config inherited from `@PSR2` and would use fixer's defaults.